src/ruby/ConditionWhile.java

```diff
@@ -0,0 +1,53 @@
+package ruby;
+
+public class ConditionWhile {
+	void displayEvenNumber(int start, int end) {
+		System.out.println("Even Number between " + start + " and " + end);
+		int index = start;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use start++ for increment?